### DISCUSSION TITLE
DMS as IMAP storage and Webmail Server?

### DISCUSSION THREAD

**Discussion by schnillerman** [docker-mailserver/docker-mailserver#3617] [Q&A]
I'm looking for a solution to move my gmail mails to a local, NAS/Docker based storage server that I can access via IMAP/webmail and integrate my gmail into that webmail server. This way I could move old mails to local IMAP folders as storage/archive and still seamlessly access gmail, all from the local webmail server.

Question is: is DMS the right application, especially if I don't plan to use it as a full blown mailserver? I don't have static IP and am not planning on hosting my very own mailserver.

I've been looking at roundcube and dovecot as an alternative.

Can someone give me a lead?

**Comment by jhenkins** [comment]
No need for a whole Postfix setup for what you want, your approach have merit. I host my own offline mail-base exactly like this, although I use Rainloop and SnappyMail instead of Roundcube. Any IMAP client will work just fine with Dovecot. I would consider the use of imapsync doing the initial big sync, and then something like offlineimap or isync to maintain the sync afterwards if you will be keeping the gmail account active.

**Reply by schnillerman** [reply]
That's exactly the kind of advice I am looking for, thanks.
Have questions though. Will come back as soon as I have some intel on the keywords you cast, if it's OK.
Thanks again!
